View/MainWindow.xaml.cs

```diff
@@ -161,11 +161,18 @@ private void Duration_ValueChanged(object sender, RoutedPropertyChangedEventArgs
         public void SetScreen(ScreenNames screenName)
         {
             MainContent.ContentTemplate = FindResource(screenName.ToString()) as DataTemplate;
+            if (screenName == ScreenNames.PlaylistScreen ||
```

Comment:
Mist hier niet een else, met `ScreenName = null` ofzo?

Reply:
Nee, want geen enkel ander screen doet dingen in de song infodatacontext

Reply:
maar dan kun je hem toch net zo goed altijd zetten?
Want het maakt nu niet uit welk scherm gezet word, die waarde in die datacontext blijft altijd staan?

Reply:
is goed